src/shell.js

```diff
@@ -270,7 +270,22 @@ if (ENVIRONMENT_IS_NODE) {
     console.error('The "worker_threads" module is not supported in this node.js build - perhaps a newer version is needed?');
     throw e;
   }
-  global.Worker = nodeWorkerThreads.Worker;
+  /**
+   * @constructor
+   * @param {string|URL} url
+   */
+  let NodeWorker = nodeWorkerThreads.Worker;
+  // Node requires data and file protocol urls to be URLs.
```

Comment:
How about "Create a polyfill for the Worker web API based on nodes `worker_threads` which is slightly different"